app/router.js

```diff
@@ -1,9 +1,28 @@
 import Ember from 'ember';
 import config from './config/environment';
+import { get } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { scheduleOnce } from '@ember/runloop';
 
 const Router = Ember.Router.extend({
+  metrics: service(),
+
   location: config.locationType,
-  rootURL: config.rootURL
+  rootURL: config.rootURL,
+
+  didTransition() {
+    this._super(...arguments);
+    this._trackPage();
+  },
+
+  _trackPage() {
+    scheduleOnce('afterRender', this, () => {
+      let page = this.get('url');
+      let title = this.getWithDefault('currentRouteName', 'unknown');
+
+      get(this, 'metrics').trackPage({ page, title });
```

Comment:
Well, this piece of code was copy-pasted from the `ember-metrics` sample. Apparently we'd need something more sophisticated